src/components/StepperVertical/StepperVertical.vue

```diff
@@ -0,0 +1,57 @@
+<template>
+  <div>
+    <StepperVerticalItem
+      v-for="step in steps"
+      :key="step"
+      :ref="step === stepWithlongestPathName ? 'longest' : null"
+      :style="`min-width:${longestWidth}px;`"
+      :step="step"
+      :dark-mode="darkMode"
+      :index="steps.indexOf(step)+1"
+      :active="activeStepPathName === step.pathName"
+      @click="handleClick(step)"
+    />
+  </div>
+</template>
+
+<script>
+import StepperVerticalItem from './StepperVerticalItem.vue';
+
+export default {
+  name: 'StepperVertical',
+  components: { StepperVerticalItem },
+  props: {
+    steps: { type: Array, required: true,
+      validator: function (value) {
+        const hasPathName = (obj) => obj.hasOwnProperty('pathName');
+        return Array.isArray(value) && value.every(hasPathName);
+      } },
```

Comment:
Added `steps` prop validator that it is an Array with every Object containing the `pathName`